proto/ligato/vpp/nat/nat.proto

```diff
@@ -178,3 +178,22 @@ message VirtualReassembly {
     // If set to true fragments are dropped, translated otherwise.
     bool drop_fragments = 4;
 }
+// Nat44EdVrfTable defines a vrf table for NAT44.
+message Nat44EdVrfTable {
+    // Id of the VRF NAT routing table
+    uint32 table_vrf_id = 1;
+    // Number of vrf_ids
+    uint32 n_vrf_ids = 2;
+    // Ids of resolving destination (tx) VRFs
+    repeated uint32 vrf_ids = 3;
+
+}
+// Nat44EdVrfRoute defines a route for NAT44 VRF tables.
+message Nat44EdVrfRoute {
+    // Id of the source vrf
+    uint32 table_vrf_id = 1;
+    // Id of the destination vrf
+    uint32 vrf_id = 2;
+    // add_del
+    bool is_add = 3;
```

Comment:
@Giluerre Please, note that Proto API is not a copy of binary API. If it was, we would simply generate it. It is a more high-level API that should aim to be more generic and defined in a way that is as much future-proof as it can.